lib/ajv.ts

```diff
@@ -31,39 +31,75 @@ class Ajv extends AjvCore {
   }
 }
 
-module.exports = exports = Ajv
-Object.defineProperty(exports, "__esModule", {value: true})
+export = Ajv
+Object.defineProperty(Ajv, "__esModule", {value: true})
+Object.defineProperty(Ajv, "default", {value: Ajv})
 
-export default Ajv
+// eslint-disable-next-line @typescript-eslint/no-namespace, no-redeclare
+declare namespace Ajv {
+  // compatibility with NodeNext
+  export {Ajv as default}
+}
+
+import * as types from "./types"
+import * as core from "./core"
+import * as compile from "./compile"
+import * as compileValidate from "./compile/validate"
+import * as vocabulariesErrors from "./vocabularies/errors"
+import * as compileRules from "./compile/rules"
+import * as typesJsonSchema from "./types/json-schema"
+import * as compileCodegen from "./compile/codegen"
+import * as runtimeValidationError from "./runtime/validation_error"
+import * as compileRefError from "./compile/ref_error"
+
+// eslint-disable-next-line @typescript-eslint/no-namespace, no-redeclare
```

Comment:
seems typescript-eslint isn't very aware of the `export import` feature in namespaces https://github.com/typescript-eslint/typescript-eslint/issues/4129